services/api/src/utils/mailer.js

```diff
@@ -34,8 +34,12 @@ async function sendTemplatedMail({ template, layout = 'layout.html', to, ...opti
     ...options,
   };
 
-  // Order is important here as interpolated variables may contain markdown.
-  const body = marked(interpolate(templateBody, vars));
+  // Two pass interpolation here allows the passed content body
+  // to also contain variables so that dynamic contenet can work.
+  let body = templateBody;
+  body = interpolate(body, vars);
+  body = interpolate(body, vars);
```

Comment:
mmm that smells like something isnt right the ... cant that lead to problems ? 
Do you have an example ? of what you try to solve here

Reply:
this is a first step to allow dynamic emails that can be edited by admins. Having a 2 pass system allows both the email templates as well as the injected comments to reference variables.

Reply:
https://github.com/bedrockio/bedrock-core/pull/169#discussion_r764920727

Reply:
it's not really an issue yet as our use case isn't fleshed out... dynamic doesn't mean the source isn't trusted. we can cross that bridge when we come to it